orcid-core/src/main/java/org/orcid/core/manager/read_only/impl/AffiliationsManagerReadOnlyImpl.java

```diff
@@ -124,8 +123,7 @@ public EmploymentSummary getEmploymentSummary(String userOrcid, Long employmentI
      * @return the list of employments that belongs to this user
      * */
     @Override
-    @Cacheable(value = "employments-summaries", key = "#userOrcid.concat('-').concat(#lastModified)")
```

Comment:
1. On some of the 2.0 managers, the **cacheable** annotation is removed,  some times it is moved to the DAO, but some other times (like this one on _orgAffiliationRelationDao.getByUserAndType(userOrcid, AffiliationType.EMPLOYMENT)_) it is just removed, this could generate an impact on the databases, please iterate over all the 2.0 managers and verify that you are not removing existing caches, and, change the 3.0 managers to be consistent, so, both managers cache things in the same way (both on manager or both on DAO)

1.1: I remember having a discussion years ago in which we said we want to have the cache at the manager level, but I don't remember the reasons why, maybe we want the manager level to take care of that and DAO layer only fetch fresh things from DB? @wjrsimpson do you remember that? is ok if we keep it like this or do you think we should audit the caches at the DAO layer and move them to the managers? if so, should we do it here or should we do it in another PR?